aptos-move/aptos-gas/src/transaction/mod.rs

```diff
@@ -45,7 +53,7 @@ crate::params::define_gas_parameters!(
         [
             maximum_number_of_gas_units: Gas,
             "maximum_number_of_gas_units",
-            2_000_000
+            200_000_000
```

Comment:
why did we increase this? where is the maximum execution units? I thought we were focusing on max execution units and then max gas units would be the additional available for storage?

Reply:
My bad, it looks like I forgot to replace this with MAX_GAS_AMOUNT, which is now gated by a cargo feature flag.